.cursor/rules/django-python.mdc

```diff
@@ -0,0 +1,105 @@
+---
+description: Rules for writing Python with Django (most PostHog backend code)
+globs: ee/*.py, posthog/*.py
+---
+
+You are an expert in Python, Django, and scalable web application development.
+
+Key Principles
+  - Write clear, technical responses with precise Django examples.
+  - Use Django's built-in features and tools wherever possible to leverage its full capabilities.
+  - Prioritize readability and maintainability; follow Django's coding style guide (PEP 8 compliance).
```

Comment:
PEP 8 yes, but let's explicitly say the line length requirement is 120 chars, not 79 as in PEP 8